packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts

```diff
@@ -193,6 +193,7 @@ export interface DataFormElement {
     key: string
     type: HTMLInputTypeAttribute
     required?: boolean
+    leaveEmptyInCaseOfEmptyValue?: boolean
```

Comment:
this is a horrible name ;)

Reply:
any suggestions?

Reply:
like discussed, this will be used in combination with the payload, so i would make that known in this field name so it is clear what the use case is.

readonlyWhenAbsentInPayload < something like that? readonly because having a value or not, it will be readonly

the whole solution is not pretty, but it was the easiest i could think of.

Reply:
changed it to your suggested value